janitor/functions.py

```diff
@@ -461,8 +461,8 @@ def coalesce(df, columns, new_column_name):
         import janitor
         df = pd.DataFrame(...).coalesce(['col1', 'col2'])
 
-    The result of this function is that we take the first non-null value across
-    rows.
+    The first example will create a new column called 'col3' with values from 'col2' inserted where values from
```

Comment:
@dendrondal, would you be kind enough to shorten the docstrings to 79 characters? I know this is currently not being caught on CI, but it's part of the code style that we're using.